### DISCUSSION TITLE
Router General enhancements

### DISCUSSION THREAD

**Discussion by BeardRedMachine** [aristanetworks/avd#3524] [Q&A] [answered]
During one of our deployments, we ran in to MTU issues and worked with TAC to identify a configuration change.  We ended up using:
`router general
software forwarding hardware offload mtu 9214`
and that solved our issue.  The current options for [router_general](https://avd.arista.com/4.3/roles/eos_cli_config_gen/docs/tables/router-general.html?h=router_general) do not have options for this, is there an easy workaround to implement this command in to our AVD deployment?

**Comment by ClausHolbechArista** [comment, marked as answer]
Hi Graham, thank you for bringing this up. The easy fix is to use `raw_eos_cli` under `platform_settings`. This way you can add the CLI you need for only the specific platforms in question.
Note that setting `platform_settings` will *replace* the defaults, so copy them from the docs and modify from there. (You find the defaults under the (+) sign on the YAML tab in the doc.

Reference:
https://avd.sh/en/devel/roles/eos_designs/docs/input-variables.html#platform-settings
